src/edu/wright/cs/raiderplanner/controller/MenuController.java

```diff
@@ -1461,9 +1463,26 @@ public void initialize(URL location, ResourceBundle resources) {
 			}
 		});
 
-		// text:
-		this.welcome = new Label(
-				"Welcome back, " + MainController.getSpc().getPlanner().getUserName() + "!");
+		/**
```

Comment:
Javadoc is only appropriate for classes, methods, class variables, etc. It does not make sense to have it in the middle of a block of code. Please remove the second asterisk in order to make this a standard comment.